scipy/stats/tests/test_multivariate.py

```diff
@@ -1635,6 +1637,191 @@ def test_haar(self):
         res = kstest(x.ravel(), uniform(-np.pi, 2*np.pi).cdf)
         assert_(res.pvalue > 0.05)
 
+
+class TestMultivariateHypergeom(object):
+    def test_logpmf(self):
+        # Ground truth value from R dmvhyper
+        vals1 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=7)
+        assert_allclose(vals1, -1.119814, rtol=1e-6)
+
+        vals2 = multivariate_hypergeom.logpmf(x=[3, 4], m=[5, 10], n=0)
+        assert_allclose(vals2, np.NAN, rtol=1e-8)
```

Comment:
```suggestion
        assert_allclose(vals2, np.NAN)
```
No difference, right? I don't mind much, but it might bother some.

Reply:
Also, consider whether this should be `NINF` instead of `NAN`. Seems to me that this is a valid urn, but there is just zero probability of observing 3 of A and 4 of B if you do `n=0` trials, right? Also, `dmvhyper` does `NINF` for this case.

Reply:
If you agree with the comment above, it would be good to add another check for `n < 0`, because that _should_ be `NAN`. You can pick `n=0` balls from the urn, but it doesn't make sense to pick `n=-1` balls from the urn.

Reply:
You are right. Fixed!